description: # gets interpreted as a comment, leading to compile errors (not a documented gcc option anyway)

--- a/src/third_party/opus/opus.gyp
+++ b/src/third_party/opus/opus.gyp
@@ -54,7 +54,6 @@
           'cflags': [
             # Suppress a warning given by opus_decoder.c that tells us
             # optimizations are turned off.
-            '-Wno-#pragma-messages',
           ],
         }],
         ['use_opus_fixed_point==0', {
